code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -46,12 +46,7 @@
 	var/can_repair_constructs = FALSE
 	var/can_repair_self = FALSE
 	var/runetype
-/*
-	For themes
-	THEME_CULT = red
-	THEME_HOLY = blue
-	THEME_WIZARD = purple
- */
+	//themes include cult red, holy blue, and wizard purple
```

Comment:
Still not a doc comment.

https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md